README.md

```diff
@@ -89,7 +89,7 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
         with:
-          token: ${{ secrets.GITHUB_TOKEN }}
+          submodules: true
```

Comment:
ah great! Did not know that now the default for this action is `github.token`

Also like the example for `submodules`. One thought:
My guess is that most teams/people do not use submodules maybe a small change like

```yaml
- name: Checkout
  uses: actions/checkout@v4
  # https://github.com/actions/checkout#usage
  # uncomment if you use submodules within the source repository
  # with:
    # submodules: true
```

would be more generic. @jellllly420  What do you think?

Reply:
OK, fixed in de2bc5e.